CHANGELOG.md

```diff
@@ -42,6 +43,16 @@
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.2.1]
+
+> Release date: TBD
+
+#### Fixed
+
+- Added mitigation for an issue where controllers may temporarily incorrectly
+  delete configuration upon gaining leadership.
```

Comment:
I think it's worth very deliberately explaining what we mean by "temporarily incorrectly delete"

```suggestion
- Added mitigation for an issue where controllers may briefly delete and
  recreate configuration upon gaining leadership while populating their
  Kubernetes object cache.
```